src/main/scala/mesosphere/marathon/event/Events.scala

```diff
@@ -40,6 +40,14 @@ object EventModule {
   final val busName = "events"
 }
 
+/** Local leadership events. They are not delivered via the event endpoints. */
+sealed trait LeadershipStateEvent
+
+object LeadershipStateEvent {
+  case object ElectedAsLeader
```

Comment:
I could imagine, that this kind of event is also helpful for external services. 
For that it should have something like the IP Address.
As SSE Listener this is would probably be the last message, that is received.